source/backend/api/Areas/Takes/Controllers/TakeController.cs

```diff
@@ -99,15 +99,15 @@ public IActionResult UpdateAcquisitionPropertyTakes(long acquisitionFileProperty
         }
 
         /// <summary>
-        /// Gets a list of takes that belong to the associated property id.
+        /// Gets a count of takes that that match a property and are not a part of an AcquisitionFile.
         /// </summary>
         /// <returns></returns>
-        [HttpGet("property/{propertyId:long}/count")]
+        [HttpGet("acquisition/{acquisitionFileId:long}/property/{propertyId:long}/count")]
         [HasPermission(Permissions.AcquisitionFileView, Permissions.PropertyView)]
         [Produces("application/json")]
         [ProducesResponseType(typeof(TakeModel), 200)]
         [SwaggerOperation(Tags = new[] { "take" })]
-        public IActionResult GetTakesCountByPropertyId(long propertyId)
+        public IActionResult GetTakesCountByPropertyId([FromRoute]long acquisitionFileId, [FromRoute]long propertyId)
```

Comment:
Why was this change necessary? you already had the:
A)all takes for the current property.
B)all takes for the current property on the current file

So if you want the count of takes that aren't part of the acquisition file, you simply subtract b from a. Am I missing something?

Reply:
@devinleighsmith the PropertyId would only let me know how many takes does the property has on all Acquisition File, but the new enhancement would need to know which Acquisition File is being view to discard the takes in that Acquisition File. 
B) is not true since propertyId could be on any given acquisition file.

Reply:
We discussed this, moving to the strategy of the frontend doing the counts. The frontend will use: 1 count endpoint as well as the actual takes response, do the subtraction, and display the final number.

Reply:
Updated